tests/main/snapshot-basic/task.yaml

```diff
@@ -29,9 +30,17 @@ execute: |
     # and is valid
     snap check-snapshot "$SET_ID"
 
+    # and is exportable
+    snap export-snapshot "$SET_ID" "${SET_ID}_snapshot_export.tar"
+    test -e "${SET_ID}_snapshot_export.tar"
+    test "$( tar -tvf "${SET_ID}_snapshot_export.tar" | wc -l )" = "3"
```

Comment:
can we also add another snippet that does the same operation again to ensure that the size of the files is the same?

Reply:
Sure, added